.github/workflows/ci.yml

```diff
@@ -1,70 +1,174 @@
+            - name: Build
+              uses: actions-rs/cargo@v1
+              with:
+                  command: build
+                  args: --release
+              env:
+                  CARGO_INCREMENTAL: 1
+            - name: Test
+              uses: actions-rs/cargo@v1
+              with:
+                  command: test
+                  args: --release
+              env:
+                  CARGO_INCREMENTAL: 1
+            - name: Compress node
+              run: cp target/release/drml . && tar cf node.tar.zst drml -I pzstd
+            - name: Upload node
+              uses: actions/upload-artifact@v2
+              with:
+                  name: node
+                  path: node.tar.zst
+    check_default_features:
+        name: Check default features
+        runs-on: ubuntu-latest
+        needs: [build_kit]
+        steps:
+            - name: Fetch latest code
+              uses: actions/checkout@v2
+            - name: Download kit
+              uses: actions/download-artifact@v2
+              with:
+                  name: kit
+            - name: Uncompress kit
+              run: tar xf kit.tar.zst -I pzstd
+            - name: Run checker
+              run: ./ci-bot default-features --project-dir ../darwinia-common
+    check_rust_code:
+        name: Check Rust code
+        runs-on: ubuntu-latest
+        steps:
+            - name: Fetch latest code
+              uses: actions/checkout@v2
+            - name: Install Rust nightly-2021-02-25 toolchain
+              uses: actions-rs/toolchain@v1
+              with:
+                  toolchain: nightly-2021-02-25
+                  target: wasm32-unknown-unknown
+                  default: true
+            - name: Run checker
+              uses: actions-rs/cargo@v1
+              with:
+                  command: check
+              env:
+                  CARGO_INCREMENTAL: 1
+    check_storage_prefix:
+        name: Check storage prefix
+        runs-on: ubuntu-latest
+        needs: [build_kit, build_node_and_internal_test]
+        outputs:
+            result: ${{ steps.run_checker.outputs.result }}
+        steps:
+            - name: Download kit
+              uses: actions/download-artifact@v2
+              with:
+                  name: kit
+            - name: Download node
+              uses: actions/download-artifact@v2
+              with:
+                  name: node
+            - name: Uncompress kit and node
+              run: tar xf kit.tar.zst -I pzstd && tar xf node.tar.zst -I pzstd
+            - id: run_checker
+              name: Run checker
+              run: |
+                  OUTPUT=$(./ci-bot storage-prefix --chain pangolin --exec ./drml)
+                  OUTPUT="${OUTPUT//'%'/'%25'}​"
+                  OUTPUT="${OUTPUT//$'\n'/'%0A'}"
+                  OUTPUT="${OUTPUT//$'\r'/'%0D'}"
+                  echo "::set-output name=result::$OUTPUT"
+            - uses: marocchino/sticky-pull-request-comment@v2
```

Comment:
This workflow could be triggered by both `push` and `pull_request`, what does this action do if it's in a `push` context?

Reply:
Set `if: github.ref != 'refs/heads/master'`